cookbook/README.md

```diff
@@ -4,6 +4,7 @@ Example code for building applications with LangChain, with an emphasis on more
 
 Notebook | Description
 :- | :-
+[agent_fireworks_ai_langchain_mongodb.ipynb](https://github.com/langchain-ai/langchain/tree/master/cookbook/agent_fireworks_ai_langchain_mongodb.ipynb) | Build an AI Agent With Memory Using MongoDB, LangChain and FireWorksAI.
```

Comment:
could we add one for mongodb-langchain-cach... as well?

Reply:
Hi @baskaryan 

A use case for caching is currently in the works.
If we can get this merged into the repo, the team will look into getting a caching component incorporated into the notebook as soon as possible.

Thanks

Reply:
@baskaryan added the entry for cache as well